examples/csv_mail_merge/index.js

```diff
@@ -0,0 +1,80 @@
+'use strict';
+
+var converter = require('json-2-csv');
+var fs       = require('fs');
+var moment   = require('moment');
+var parse    = require('csv-parse');
+
+var lobFactory = require('../../lib/index.js');
+var Lob        = new lobFactory('test_0dc8d51e0acffcb1880e0f19c79b2f5b0cc');
+
+var inputFile = fs.createReadStream(__dirname + '/input.csv');
+var letterTemplate = fs.readFileSync(__dirname + '/letter_template.html').toString();
+
+var companyInfo = {
+  name: 'Deluxe Virgina Realty',
+  address_line1: '185 Berry St.',
+  address_line2: 'Ste 170',
+  address_city: 'San Francsisco',
+  address_state: 'CA',
+  address_zip: 94107,
+  address_country: 'US'
+}
+
+var parser = parse({columns: true}, function (err, data) {
```

Comment:
@dmlittle 
`{columns: true}` is this telling the parser to skip the first line in the CSV?